src/main/java/codeu/model/store/basic/ConversationStore.java

```diff
@@ -96,6 +97,48 @@ public Conversation getConversationWithTitle(String title) {
     return null;
   }
 
+  /** Find and return a List of Conversations that the user is able to access and chat in */
+  public List<Conversation> getConversationsForUser(String username) {
+    // TODO: decide whether or not this is needed (currently filtering conversations in conversations.jsp)
+    List<Conversation> userConversations = new ArrayList<>();
+
+    for (Conversation conversation : conversations) {
+      if (conversation.isNormalConversation()) {
+        System.out.println("Normal: " + conversation.getTitle());
```

Comment:
Were these print statements just meant for debugging or do you want the application to acutally log these messages. If so, maybe use [GAE logging](https://cloud.google.com/appengine/docs/standard/java/logs/), or you could just get rid of these here since they don't seem to be doing much

Reply:
They were meant for debugging and I forgot to remove them, thanks for pointing this out. I'll remember to use GAE logging if I want to log anything in the future!